client/src/main/java/plugin/api/API.java

```diff
@@ -309,4 +313,37 @@ public static void SendMessage(String message) {
     public static void RequestNewSong() {
         SoundPlayer.sendTrackEndPacket();
     }
+    public static JagString getHexFromRGB(int red, int green, int blue) {
```

Comment:
You need to follow API case conventions, namely this should be GetHexFromRgb, and the method below this one should be CalcEntityScreenPos. API methods also have to be well documented with clearly defined arguments, which the CalcEntityScreenPos method heavily lacks.